<?xml version="1.0"?>
<launch>
  <node pkg="initialpose_publisher" type="initialpose_publisher.py" name="initialpose_publisher" >
    <param name="map_frame_id" value="map" />
    <param name="base_frame_id" value="base_link" />
    <param name="topic_name" value="/hsas_global/start" />
    <param name="hz" value="20" />
  </node>
</launch>
